SimpleRss/SimpleRss/Models/Feed.swift

```diff
@@ -8,9 +8,18 @@
 
 import Foundation
 
-struct Feed {
-    let title: String
-    let pubDate: String
-    let picUrl: String
-    let description: String
+class Feed {
+    var isExpanded: Bool
```

Comment:
Please, avoid adding isExpanded flag in data model. I would consider creating viewModel for cell for such cases.
Look at Feed as a response from server, which must be immutable. We can pass the response to other view controllers that are not aware of purpose of the flag.